source/lib/config/user_data/data_user_data

```diff
@@ -58,6 +58,7 @@ user=${USER_NAME}
 autorestart=true
 redirect_stderr=true
 stdout_logfile=/var/log/supervisor/historical.log
+stopwaitsecs=30
```

Comment:
Can we put this  in `common_user_data` so we don't have to repeat it for each individual service?

Reply:
the supervisord config for each druid component is generated in individual user_data. They are slightly different, bringing these customisations into common_user_data would defeat the purpose of having `common`.

I'd keep it like this for now.